apps/mull-ui/src/app/components/navigation/sub-nav-bar/sub-nav-bar.scss

```diff
@@ -10,16 +10,19 @@
 
   // Needed since SubNavBar can't be positioned at the end of the page, since it must be part of the home route
   z-index: 2;
+
+  &.with-header {
+    top: 5.75rem;
```

Comment:
This value should be `$top-nav-bar-height-mobile` + `$top-nav-header-height-mobile` if I understand correctly. This sub nav bar should be put under those two other components. just like how's its currently done on desktop